addons/l10n_in_edi/models/account_edi_format.py

```diff
@@ -408,15 +408,23 @@ def _l10n_in_edi_generate_invoice_json(self, invoice):
         if is_overseas:
             json_payload.update({
                 "ExpDtls": {
-                    "ShipBNo": invoice.l10n_in_shipping_bill_number or "",
-                    "ShipBDt": invoice.l10n_in_shipping_bill_date
-                       and invoice.l10n_in_shipping_bill_date.strftime("%d/%m/%Y") or "",
-                    "Port": invoice.l10n_in_shipping_port_code_id.code or "",
                     "RefClm": tax_details_by_code.get("igst") and "Y" or "N",
                     "ForCur": invoice.currency_id.name,
                     "CntCode": saler_buyer.get("buyer_details").country_id.code or "",
                 }
             })
+            if invoice.l10n_in_shipping_bill_number:
```

Comment:
@jva-odoo @nia-odoo If you give shipping bill number, should not you also give the date or is each of them optional on its own?

Reply:
Hello @jco-odoo, 
Both of them are individually optional.